pom.xml

```diff
@@ -22,7 +22,7 @@
 
     <groupId>org.apache.xtable</groupId>
     <artifactId>xtable</artifactId>
-    <name>xtable</name>
+    <name>incubator-xtable</name>
```

Comment:
Q: Is this required ? The guidelines say it is. 
https://incubator.apache.org/guides/releasemanagement.html 
The [Incubator policies](https://incubator.apache.org/incubation/Incubation_Policy.html#Releases) applies two additional constraints to podlings for their releases. They are repeated here for clarity only. - Release artifacts must include incubating in the final file name - Release artifacts must include one of two [disclaimers](https://incubator.apache.org/guides/branding.html#disclaimers)

Reply:
This is required, peeked into other projects as well.